estmator_project/est_quote/models.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.contrib.auth.models import User
+from est_client.models import Client
+
+
+@python_2_unicode_compatible
+class Category(models.Model):
+    name = models.CharField(max_length=256)
+
+    def __str__(self):
+        return 'Category: {}'.format(self.name)
+
+
+@python_2_unicode_compatible
+class Product(models.Model):
+    # Inputs
+    category = models.ForeignKey(Category)
+    name = models.CharField(max_length=256)
+
+    # Piece Multipliers
+    mins_piece = models.FloatField()
+    mult_dollies = models.FloatField()
+    m_cart = models.FloatField()
+    l_cart = models.FloatField()
+    p_cart = models.FloatField()
+    s_pack = models.FloatField()
+
+    def __str__(self):
+        return 'Product: {}'.format(self.name)
+
+
+class QuoteModifiers(models.Model):
+    # Location multipliers
+    street_load = models.FloatField()
+    midrise_elev_std = models.FloatField()
+    midrise_elv_frt = models.FloatField()
+    highrise = models.FloatField()
+    stairs = models.FloatField()
+    lng_psh = models.FloatField()
```

Comment:
Again, abbreviated names are tempting, but almost never end up working out as well as you'd like.